agent/exporter/src/main/java/com/microsoft/applicationinsights/agent/Exporter.java

```diff
@@ -528,6 +528,15 @@ private void exportRequest(SpanData span) {
         requestData.setId(span.getSpanId());
         requestData.getContext().getOperation().setId(span.getTraceId());
 
+        String locationIp = attributes.get(SemanticAttributes.HTTP_CLIENT_IP);
+        if (locationIp == null) {
+            // only use net.peer.ip if http.client_ip is not available
+            locationIp = attributes.get(SemanticAttributes.NET_PEER_IP);
+        }
+        if (locationIp != null) {
```

Comment:
i checked customMetrics table, it has the value of "0.0.0.0".  when it's empty, does Breeze assign this value to client_ip?

Reply:
Breeze masks client IP by default, you have to manually override:
https://docs.microsoft.com/en-us/azure/azure-monitor/app/ip-collection?tabs=net#storing-ip-address-data